src/pizzeria/Menu.java

```diff
@@ -0,0 +1,11 @@
+package pizzeria;
+
+public enum Menu {
```

Comment:
You can use this enum when passing a Pizza name.

Reply:
I deleted that enum class, and for passing Pizza name used List